### DISCUSSION TITLE
Include page screenshot in web scraper output

### DISCUSSION THREAD

**Discussion by pjamessteven** [langgenius/dify#3734] [Suggestion]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Pleas do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing?

When using the webscraper, sometimes the content I am after is text in an image, which means that I would have to iterate over every image on the page and perform OCR on each image to get the required content. This makes the workflow really complex.

### 2. Describe the feature you'd like to see

If the web scraper could return the full rendered page as an image, alongside the raw HTML, I could then perform OCR on the whole page with GPT to get the required content. 

### 3. How will this feature improve your workflow or experience?

This would make the web scraper much more useful and versatile.

### 5. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by wolfei666** [comment]
Hi, have you resolved this issue? I am facing the same issue.
